examples/build.gradle

```diff
@@ -46,9 +46,9 @@ task testExamples {
                 exec {
                     workingDir project.projectDir
                     if (Os.isFamily(Os.FAMILY_WINDOWS)) {
-                        commandLine 'cmd', '/c', "${ballerinaDist}/bin/bal.bat test --offline ${example} && exit %%ERRORLEVEL%%"
+                        commandLine 'cmd', '/c', "${ballerinaDist}/bin/bal.bat test --native --offline ${example} && exit %%ERRORLEVEL%%"
```

Comment:
Suggested to have the flag only when `balNativeTest` property is set

```gradle
def nativeFlag = ""

task testExamples {
    if (project.hasProperty('balNativeTest')) {
         nativeFlag = '--native'
     }
    doLast {
        examples.each { example ->
            try {
                exec {
                    workingDir project.projectDir
                    if (Os.isFamily(Os.FAMILY_WINDOWS)) {
                        commandLine 'cmd', '/c', "${ballerinaDist}/bin/bal.bat test ${nativeFlag} --offline ${example} && exit %%ERRORLEVEL%%"
                    } else {
                        commandLine 'sh', '-c', "${ballerinaDist}/bin/bal test ${nativeFlag} --offline ${example}"
                    }
                }
            } catch (Exception e) {
                println("Example '${example}' Build failed: " + e.message)
                throw e
            }
        }
    }
}
```

Reply:
My bad, sorry. Updated the PR.